bench/normal-setup.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const authDirective = `directive @auth(
+  requires: Role = ADMIN,
+) on OBJECT | FIELD_DEFINITION
+
+enum Role {
+  ADMIN
+  REVIEWER
+  USER
+  UNKNOWN
+}`
+
+const schema = `
+  ${authDirective}
```

Comment:
Why are we duplicating this in two places? Can we avoid it?

Reply:
when you say duplicating, are you referring to this code being used in both `normal-with-auth.js and normal-wihout-auth.js`? Or the fact that it's very similar to code in the `examples` (which I could benchmark against instead to be fair)?

Reply:
No. I mean the `auth` is being passed in inside the schema and as an option of the module. We might be able to skip it entirely? Maybe we can just pass in the name of the directive to fetch from the parsed schema?

Reply:
Yeah that makes sense as we only use the name anyway and directive names are unique within the schema - it also has the benefit of preventing users from defining multiple directives within the definition they are providing! I'll get on that asap!